### ISSUE TITLE
Move value filter not rendering properly during move transition in obs 28

### ISSUE THREAD

**Issue by duncte123** [exeldro/obs-move-transition#129]
First things first, this worked fine in obs version 27 and I have not changed anything about the configuration after I upgraded the plugin.
Here's a video of the issue:  https://www.twitch.tv/videos/1609288528

Transition config (one way as the two ways are essentially copies with slight alterations to what does not have the transform applied)
![](https://i-illegally.entered.space/d0krWcgtmI.png)
![](https://i-illegally.entered.space/naRbqoxfN5.png)

filter configurations:
![](https://i-illegally.entered.space/hfuspesv4q.png)
![](https://entered.space/OZaIgGTyRq.png)

And filter number two
![](https://going-g.host/G6ikDZZuu0.png)
![](https://leeks.life/MdQyChaTES.png)

Not sure what I did to break the plugin like that, I am able to provide logs if needed

**Comment by flexiondotorg** [comment]
@duncte123 I have experienced a similar issue with recent versions of Move Transition. I can reproduce it with OBS 27 as well. Previously working transitions now jump about, and I've not changed the transition configurations.

I've found that rolling back to Move Transition 2.6.1 (in OBS 27 and OBS 28) restores the expected/prior correct behaviour of smooth transitions.

**Comment by flexiondotorg** [comment]
The change in behaviour I am observing was introduced in this commit:

https://github.com/exeldro/obs-move-transition/commit/908cf8a635806344801e2e53973bc611b27dfdbb

**Comment by exeldro** [comment]
Can you test if this build fixes the issue for you?
https://github.com/exeldro/obs-move-transition/actions/runs/3210655744

**Comment by flexiondotorg** [comment]
I compiled from using a checkout of https://github.com/exeldro/obs-move-transition/commit/f14c71d12abfe2897a57a5eaee930e576eb599e9

Sadly, I am still seeing jumpy transitions using the new build when compared to a 2.6.1.

## Clip 2.6.1 (Smooth)

https://user-images.githubusercontent.com/304639/194753111-cf5444f3-fa78-4d33-a952-8d8fd3ccb6e6.mp4

## Clip f14c71d12abfe2897a57a5eaee930e576eb599e9 (Jumpy)

https://user-images.githubusercontent.com/304639/194753125-2c3e448e-91ac-4858-9510-2958f13d9d6c.mp4

## Scenes

Here are the sources for the two scenes

### Scene 1

![Scene 1](https://user-images.githubusercontent.com/304639/194752892-3214c1a9-ae3c-4cb2-aa28-5a725992e4d5.png)

### Scene 2

![Scene 2](https://user-images.githubusercontent.com/304639/194752896-e771f53e-1c31-4777-aa34-d32871aabca4.png)

And here are the Move Transition properties

![Properties 1](https://user-images.githubusercontent.com/304639/194752879-25d35a37-bbf3-4eb1-9a18-e5d3c72a769d.png)

![Properties 2](https://user-images.githubusercontent.com/304639/194752888-1dd596b4-b6c3-42fb-bdcd-087aae8aa807.png)

**Comment by duncte123** [comment]
Same thing over here with that build

https://user-images.githubusercontent.com/8843386/194900902-822a1898-cb34-45c4-b981-e50879cee41d.mp4

**Comment by exeldro** [comment]
In the latest master I added a setting that allows you to disable matching nested sources and groups
![image](https://user-images.githubusercontent.com/5457024/194913584-2070aeed-cc0a-4900-8abf-99841fd079d0.png)
Can you try and disable that?

**Comment by duncte123** [comment]
Yup, that was it by disabling that everything works as expected again.
Thank you <3

(leaving the issue open for Martin to confirm if that worked for them as well)

**Comment by flexiondotorg** [comment]
Thanks @exeldro :+1: 

I've built from source using https://github.com/exeldro/obs-move-transition/commit/9cc2b1bea37da3b1618a7cbeee1ff0e420a47faa and can confirm that smooth transitions are returned :slightly_smiling_face: 

It doesn't matter if *Match Nested Scenes* is ticked or unticked in my case, both appear to restore the behaviour I observed in 2.6.1.
